src/test/kotlin/org/partiql/scribe/utils/Functions.kt

```diff
@@ -0,0 +1,23 @@
+package org.partiql.scribe.utils
+
+import org.partiql.spi.function.FnOverload
+import org.partiql.spi.function.Parameter
+import org.partiql.spi.types.PType
+
+/**
+ * Utility class for test function definitions.
+ */
+class Functions {
+    companion object {
+        val scalarSplit =
```

Comment:
Maybe document via comment or naming that this is an overload defined explicitly for testing that transpilation of UDFs is working?